Vision/Vision.bs

```diff
@@ -23,28 +23,15 @@ Status Text:
 Boilerplate: omit conformance
 </pre>
 
-# Intention # {#intention}
+# Purpose of this Document # {#purpose}
 
 	This document is an articulation
-	of W3C’s mission, values, purpose, and principles;
```

Comment:
reflects change to the document structure.

Reply:
I kinda liked the original version. It doesn't mirror the document structure, but I think it gets more at the heart of what this document is trying to do. E.g. we're articulating our vision of the Web because this is how we express our values and purpose, but the _purpose_ of the document is to articulate or values and purpose.

Reply:
I agree with fantasai. I mentioned at the AB F2F discussion that I really admire the previous Vision for putting into words the things I've heard Tim say over many years.  I noted I never saw Tim put all these thoughts down all together in this way.  That consolidation or articulation of Tim's Vision was great, and I wouldn't want to lose it.  

Tim acted for many years as a trusted voice for his understanding for the potential of web technologies, and the importance of bearing in mind their impact on and for humanity (his vision). Many who joined the W3C were in alignment with and willing to come together to continue and expand positive and innovative ideas for the Web. As Tantek and others have noted, as W3C has moved to a Director-free status, we no longer have Tim as this one voice. Therefore to articulate and expand W3C's Vision for the web and web technologies is important so the organization is shown have a deep understanding on where we are heading and why. 

I appreciate the paragraph rewrites in the intro includes many of the points in the previous version. However, I would note that I think the bullet point list was incredibly effective for messaging and wonder if that could be brought back and/or refined.

[[3. Vision for the World Wide Web

The Web is for [all humanity](https://www.w3.org/TR/ethical-web-principles/#allpeople).
The Web is designed for the [good of its users](https://www.w3.org/TR/ethical-web-principles/#noharm).
The Web must be [safe for its users](https://www.w3.org/TR/ethical-web-principles/#privacy).
There is [one interoperable world-wide Web](https://www.w3.org/TR/ethical-web-principles/#oneweb).]]

Reply:
(Note that Amy's comment was about retaining the bullet list for "vision of the Web", not this line.